### ISSUE TITLE
Sidebar menu opened by tab + enter does not receive focus

### ISSUE THREAD

**Issue by andrehgdias** [tutao/tutanota#7118]
**Describe the bug**
In a single or two-column layout, if you try opening and navigating through your sidebar menu (index, drafts, etc) using Tab key you can't focus on the menu itself so the focus stays in the email list even though the sidebar is open and blocking the view. 

**To Reproduce**
1. Go to the mail list (or any other page) while in a single or two-column layout
2. Press Tab until you highlight the hamburger menu icon and press Enter
3.  Try tabbing two navigate between the menu options.

**Expected behavior**
Navigate through the menu options

### Test Notes
- [ ] On single and two-columns layout, when opening sidebar through hamburg menu, we should be able to tab inside it.
- [ ]  Tabbing anywhere else on the application should **not** be allowed **while** the sidebar is open.

**Comment by charlag** [comment]
It does work now
